public/scss/utils.scss

```diff
@@ -217,3 +217,22 @@ body.darkBG [data-tooltip]:hover::after {
   border-color: #2dd8a3;
   color: #2dd8a3;
 }
+
+.clipboard span.clickable,
+.clipboard span.alert-success {
```

Comment:
Though it likely won't affect anything because it is nested here, let's avoid modifying the Boostrap `alert-success` class. Add a custom class if necessary.

Reply:
this is done